GovUk.Frontend.Umbraco/Views/Partials/GOVUK/GovUkTaskList.cshtml

```diff
@@ -20,6 +20,7 @@
         {
             var hint = GovUkTypography.Apply(task.Content.Value<string>("hint"), new TypographyOptions { RemoveWrappingParagraph = true });
             var link = task.Content.Value<Link>("link")?.Url;
+            var statusText = GovUkTypography.Apply(task.Content.Value<string>("statusText"), new TypographyOptions { RemoveWrappingParagraph = true });
```

Comment:
I would expect this to be a plain text field, in which case you don't need to apply GOV.UK typography. Unless you think there is a genuine use case for including HTML?